test/DynamoCoreTests/CustomNodes.cs

```diff
@@ -744,13 +744,13 @@ public void TestCustomNodeLacing()
             ViewModel.OpenCommand.Execute(dynFilePath);
```

Comment:
`ArgumentLacing` cannot be set directly now, so go through backdoor API `ModelBase.UpdateValue` method.